src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -24,6 +24,25 @@ export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
   public hass?: HomeAssistant;
   private _config?: Config;
   private _configEntities?: EntityConfig[];
+  private _initializing?: boolean;
+
+  protected constructor() {
+    super();
+    this._initializing = true;
+  }
+
+  public setConfig(config: Config): void {
+    this._config = { type: "glance", ...config };
+    this._configEntities = processEditorEntities(config.entities);
+  }
+
+  protected async update(changedProps) {
```

Comment:
same return type here `Promise<void>`

Reply:
This is where you should use the suggestion button (most left button on the edit buttons for a comment)

```suggestion
  protected async update(changedProps: PropertyValue): Promise<void> {
```